infra/gravity/log.go

```diff
@@ -0,0 +1,20 @@
+package gravity
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	sshutils "github.com/gravitational/robotest/lib/ssh"
+)
+
+func init() {
+	log.SetFlags(0)
+}
+
+func Logf(t *testing.T, prefix string) sshutils.LogFnType {
+	return func(format string, args ...interface{}) {
+		t.Logf(format, args...)
+		log.Printf(fmt.Sprintf("[%s] %s", prefix, format), args...)
+	}
+}
```

Comment:
as discussed, will stay as is: in non-verbose mode, will just be equal to t.Logf and thus will log line numbers correctly.